package.json

```diff
@@ -38,7 +40,9 @@
     "axios": "^0.18.0",
     "bootstrap": "^4.3.1",
     "express": "^4.16.4",
+    "form-data": "^2.3.3",
```

Comment:
@leksyib why do we need this?

Reply:
@9jaswag  The actual FormData API was giving me some issues, also I had to get `boundary` while generating a request header but the FormData API wasn't giving the boundary for the header.

Reply:
Also why are we using the `function` keyword?

Reply:
I was flagged seriously by hound for using `const` but I think it has been fixed. I'll make changes now.